harpy/bin/molecule_coverage.py

```
@@ -0,0 +1,95 @@
+            IDX_START = idx
+        if val.strip() == "end":
+            IDX_END = idx
+    if IDX_CONTIG is None or IDX_START is None or IDX_END is None:
+        print("Error: Required columns 'contig', 'start', or 'end' not found in header", file=sys.stderr)
+        sys.exit(1)
+    while True:
+        line = statsfile.readline()
+        if not line:
+            if LASTCONTIG:
+                # write the last contig to file
+                write_coverage(coverage, LASTCONTIG)
```

Ruff

Undefined name `coverage`

(F821)